src/containers/edit-profile/professional-category/ProfessionalCategory.tsx

```diff
@@ -0,0 +1,155 @@
+import Box from '@mui/material/Box'
+import Divider from '@mui/material/Divider'
+import IconButton from '@mui/material/IconButton'
+import Typography from '@mui/material/Typography'
+import Tooltip from '@mui/material/Tooltip'
+import { FC } from 'react'
+import { useTranslation } from 'react-i18next'
+import AppButton from '~/components/app-button/AppButton'
+import { ButtonVariantEnum } from '~/types'
+import DeleteIcon from '@mui/icons-material/Delete'
+import useConfirm from '~/hooks/use-confirm'
+import { styles } from './ProfessionalCategory.styles'
+import {
+  OpenProfessionalCategoryModalHandler,
+  ProfessionalCategoryWithActivationControls
+} from '~/types'
+
+interface ProfessionalCategoryProps {
+  item: ProfessionalCategoryWithActivationControls
+  openProfessionalCategoryModal: OpenProfessionalCategoryModalHandler
+}
+
+const ProfessionalCategory: FC<ProfessionalCategoryProps> = ({
+  item,
+  openProfessionalCategoryModal
+}) => {
+  const { t } = useTranslation()
+  const { openDialog } = useConfirm()
+
+  const handleActivateButtonClick = () => {
+    // TODO: handle activation logic
+  }
+
+  const handleDeactivateButtonClick = () => {
+    // TODO: handle deactivation logic
+  }
+
+  const handleDeleteButtonClick = () => {
+    openDialog({
+      title: t(
+        'editProfilePage.profile.professionalTab.deleteCategoryModal.title'
+      ),
+      message: t(
+        'editProfilePage.profile.professionalTab.deleteCategoryModal.description'
+      ),
+      cancelButton: t(
+        'editProfilePage.profile.professionalTab.deleteCategoryModal.cancelBtn'
+      ),
+      confirmButton: t(
+        'editProfilePage.profile.professionalTab.deleteCategoryModal.submitBtn'
+      ),
+      sendConfirm: () => {
+        // TODO: handle deleting category
+      }
+    })
+  }
+
+  const CardItem = ({
+    label,
+    children
+  }: {
+    label: string
+    children: string
+  }) => {
+    return (
+      <Box sx={styles.card.item}>
+        <Typography component='dt' sx={styles.card.item.label}>
+          {label}:
+        </Typography>
+        <Typography component='dd' sx={styles.card.item.value}>
+          {children}
+        </Typography>
+      </Box>
+    )
+  }
+
+  const ToolbarButtons = item.isActivated ? (
+    <>
+      <AppButton
+        onClick={() => openProfessionalCategoryModal(item)}
+        variant={ButtonVariantEnum.ContainedLight}
+      >
+        {t('editProfilePage.profile.professionalTab.editCategoryBtn')}
+      </AppButton>
+      <Tooltip
+        arrow
+        placement='right'
+        title={
+          <Typography sx={styles.toolbar.deactivateButtonTooltip}>
+            {t(
+              `editProfilePage.profile.professionalTab.deactivateCategoryBtn${
+                !item.isActivationBlocked ? 'Enabled' : 'Disabled'
```

Comment:
I guess, you should use translation for 'Enabled' and 'Disabled'

Reply:
No, i should have two separate translations: `editProfilePage.profile.professionalTab.deactivateCategoryBtn.EnabledTooltip` and `editProfilePage.profile.professionalTab.deactivateCategoryBtn.DisabledTooltip`. I just used ternary expression here to avoid duplication

Reply:
my bad, didn`t see it